types/index.d.ts

```diff
@@ -88,11 +88,12 @@ export interface ResultsResponse<T> {
   count: number;
   page: number;
   pages?: {
-    [index: number]: {
-      start: number;
-      end: number;
-    };
+      [index: number]: {
```

Comment:
can you preserve the original indentation?  The `prettier` config should limit it to 2 spaces

Reply:
yes of course